lib/metadata.js

```diff
@@ -71,19 +70,15 @@ Metadata.prototype.updateOidcMetadata = function updateOidcMetadata(doc, next) {
 
   const self = this;
 
-  self.oidc = objectTransform({
-    source: doc,
-    map: {
-      id_token_signing_alg_values_supported: 'algorithms',
-      authorization_endpoint: 'auth_endpoint',
-    },
-    pick: [
-      'end_session_endpoint',
-      'issuer',
-      'token_endpoint',
-      'userinfo_endpoint',
-    ],
-  });
+  var oidc = {};
+  oidc.algorithms = doc.id_token_signing_alg_values_supported;
```

Comment:
I changed the auth_endpoint to authorization_endpoint everywhere in the second commit.